pkg/lib/indexer/indexer.go

```diff
@@ -140,15 +151,27 @@ func (i ImageIndexer) DeleteFromIndex(request DeleteFromIndexRequest) error {
 		return err
 	}
 
-	databasePath, err := i.ExtractDatabase(buildDir, request.FromIndex, request.CaFile, request.SkipTLS)
+	fromIndex := image.SimpleReference(request.FromIndex)
+
+	// Get the old index image's dbLocationLabel to find this path
+	labels, err := i.ImgRegistry.Labels(context.TODO(), fromIndex)
```

Comment:
Nit: is it possible to add a `context.Context` parameter to this function signature or do we have an API guarantee to maintain? It'd be nice to pipe a real context all the way in from cobra.